README.md

```diff
@@ -1,14 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio: Setup and Mobile First
+# Portfolio: Mobile and Browser
 
-> The start of the Portfolio project, possessing only the toolbar and headline sections. Now expanded with Works and about sections.
+> The Portfolio project, updated with a functional browser version and responsive design.
 
 
 ## Built With
 
 - HTML/CSS
 
+## Getting Started
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.